resources/js/pages/social/components/ManageApplicationsModal.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <Loading v-if="loading" />
+    <div v-else>
+        <template v-for="application in applications" :key="application.id">
+            <p>
+                {{`${application.username}, applied on: ${application.applied_at}`}}
+                <button class="m-1" @click="rejectApplication(application)">
+                    {{ $t('reject-group-application') }}
+                </button>
+                <button class="m-1" @click="acceptApplication(application)">
+                    {{ $t('accept-group-application') }}
+                </button>
+            </p>
+        </template>
+    </div>
+</template>
+
+<script setup>
+import {useGroupStore} from '/js/store/groupStore';
+import {ref, onMounted, computed} from 'vue';
+const groupStore = useGroupStore();
+
+const emit = defineEmits(['reloadGroups']);
+const loading = ref(true);
+onMounted(() => {
+    load();
+});
+async function load() {
+    await groupStore.fetchApplications(props.group);
+    loading.value = false;
```

Comment:
You're never setting this value to 'true'. When reloading the modal, you can set it to true, so the user knows it's loading. I have loading plans in the future, like on a button, but for now just set it to true in the start of the function.